### PR TITLE
Track pmap timeout through a fuse process

### PR DESCRIPTION
**Pull request by keynslug**
This should make code simpler and help with potential races and leftovers in mailbox.

### CODE REVIEW

**Review by ciiol — DISMISSED** [review]
👍

**Review by dinama — DISMISSED** [review]
Проблем не увидел. При сложноватой логике - отсутствие описания взаимодействия процессов усложняет чтение.  Умолчальный таймаут можно в define вынести.

File: src/genlib_pmap.erl [hunk 1]
```diff
@@ -13,6 +13,8 @@
     proc_limit => pos_integer()
 }.
 
+ -dialyzer({nowarn_function, [spawn_fuse/1]}).
```
Comment (antibi0tic) [inline src/genlib_pmap.erl]: :crying_cat_face: 
Why do we have to mute this instead of specifying `no_return()` in spec?

Comment (keynslug) [reply src/genlib_pmap.erl]: I tried it and it didn't seem to be enough. However turns out there's a simple trick to stop dialyzer from complaining here, see https://github.com/rbkmoney/genlib/pull/28/commits/273327104e85a595b89d1f8a58605756e188ab38.

**Review by ciiol — APPROVED** [review]

**Pull request merged**